test-worker/test/asset-compute/worker/png-webp/validate

```diff
@@ -0,0 +1 @@
+../validate-image.sh
```

Comment:
I'm slightly surprised by the rendition, its quite '"pixelized".

See around the Earth atmosphere in the renditions:
![image](https://user-images.githubusercontent.com/60901087/95804067-92b74e80-0cb6-11eb-9798-829c67e7d77c.png)

It's because of the compression - but I don't know if this would be acceptable... But, on the other hand, we don't really have a choice here (except offering lossless webp as option).
A good thing would be to compare (losslessly compressed) tiff and (lossless) webp to see a) if there is pixelization or not and b) if the lossless webp is smaller than the tiff.

Reply:
It depends on the input file. These test files were jpgs first then turned into png. It can look better for others, and there is the `quality` option too.

In the end, we just use the standard google webp library (through imagemagick) and don't redefine the look of it.